keyboards/handwired/hillside/52/keymaps/default_dot_c/keymap.c

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Michael McCoyd (@mmccoyd)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _QWERTY = 0,
+    _SYM,
+    _NUM,
+    _ADJUST,
+};
+
+#define xxxxxxx KC_NO
```

Comment:
Interesting key codes stand out well from _______.
They do not stand out so well from XXXXXXX, because they are both the same height. 
They do stand out pretty well from FRED xxxxxxx. XXXXXXX SALLY.

Reply:
I've looked at this both ways and it is much less readable with capital Xs.
I can change to just using the built in define, but it is less clear.